src/notehead.ts

```diff
@@ -18,21 +18,20 @@ function L(...args: any[]) {
 }
 
 export interface NoteHeadStruct extends NoteStruct {
-  glyph_font_scale?: number;
-  slashed?: boolean;
-  style?: ElementStyle;
-  stem_down_x_offset?: number;
-  stem_up_x_offset?: number;
-  custom_glyph_code?: string;
-  x_shift?: number;
-  line?: number;
-  stem_direction?: number;
-  displaced?: boolean;
-  //  duration: string;
+  glyph_font_scale: number;
```

Comment:
@rvilarl Are you going to try the all optional ? plus the `Required<NoteStruct>` with this PR? If you need help with testing or implementation, let me know and I can work off your branch.